src/blameandshame/base.py

```diff
@@ -77,6 +78,11 @@ def from_disk(path: str) -> 'Project':
     def __init__(self, repo: git.Repo) -> None:
         self.__repo: git.Repo = repo
         self.update()
+        self.blame_info_dict: Dict[Tuple[str, str],
```

Comment:
I would prefer that we keep this private. I.e., `blame_info_dict` should be `__blame_info_dict`. If we need to expose it, we can use a `@property`.